Tests/conf.json

```diff
@@ -5099,6 +5099,11 @@
             ],
             "playbookID": "GoogleSheets-Test",
             "fromversion": "6.1.0"
+        },
+        {
+            "integrations": "BitSight for Security Performance Management",
+            "playbookID": "BitSight for Security Performance Management-Test",
+            "fromversion": "6.0.0"
         }
```

Comment:
```suggestion
```
Please remove the test-playbook from the file

Reply:
Any specific reason for removing this?

Reply:
We don't add partner test playbook to this file